### DISCUSSION TITLE
Welcome to helix-home Discussions!

### DISCUSSION THREAD

**Discussion by tripodsan** [adobe/helix-home#173] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by patelpk** [comment]
I keep an eye on the progress of helix and as topics come up potentially provide input.  My goal is to make sure what is being done on helix and how it impacts other 3rd party capabilities that are typically needed in customer projects are considered. (Things like Algolia)

Piyush
